docs/general/builtins/window-functions.rst

```diff
@@ -318,10 +318,8 @@ then it can only add clauses from the referenced window, but not overwrite them.
    cr> SELECT
    ...   x,
    ...   LAST_VALUE(x) OVER (w ORDER BY x)
-   ... FROM unnest(
-   ...      [1, 2, 3, 4],
-   ...      [1, 1, 2, 2]) as t(x, y)
-   ... WINDOW w AS (PARTITION BY y)
+   ...   FROM (VALUES (1, 1), (2, 1), (3, 2), (4, 2)) as t(x, y)
+   ...   WINDOW w AS (PARTITION BY y)
```

Comment:
The `FROM` and `WINDOW` have been indented too much, can you de-indent them to match the level of `SELECT` above.

Reply:
Not a required change, but just a suggestion. Maybe `FROM (VALUES (1, 1), (2, 1), (3, 2), (4, 2)) as t(x, y)` could be written as

```
FROM (VALUES 
     (1, 1), 
     (2, 1), 
     (3, 2), 
     (4, 2) ) as t(x, y)
```

to be a little clearer.